src/azure-cli/azure/cli/command_modules/acr/_errors.py

```diff
@@ -117,6 +117,11 @@ def format_error_message(self, *args):
     "Access to registry '{}' was denied. Response code: {}. Please try running 'az login' again to refresh permissions."
 )
 
+CONNECTIVITY_TOOMANYREQUESTS_ERROR = ErrorClass(
+    "CONNECTIVITY_TOOMANYREQUESTS_ERROR",
+    "Too many requests, access to registry '{}' was denied. Response code: 429. Please wait a moment before trying again."
```

Comment:
nit: we don't need to expose the response code to the customers.
```suggestion
    "Too many requests, access to registry '{}' was denied. Please wait a moment before trying again."
```